docs/guide/user-guide.md

```diff
@@ -1 +1,109 @@
+
+To use the DeepESDL JupyterLab environment, navigate to
+<https://deep.earthsystemdatalab.net/> with a web browser (a recent version of
+Firefox, Chrome, or Safari is recommended).
+
+DeepESDL uses a GitHub to authenticate, so if you are already registered as a 
+DeepESDL user, please use your GitHub account to log in. 
+If your Jupyter server is not already running, you may be presented
+with a menu of user JupyterLab profiles to use for your session; there might be 
+one or more JupyterLab profiles to choose from, depending on the computational 
+resources needs of your team. Please select an adequate JupyterLab profiles for 
+your current task; it might not always require the strongest computational 
+resources available.
+After choosing your environment, you will see a progress bar appear for a few 
+seconds while it is started for you. 
+The JupyterLab interface will then appear in your web browser, ready for
+use.
+
+
+### Choosing a JupyterLab profile
+
+If you don't already have a running DeepESDL Jupyter session, you will be 
+presented with a list of one or more available user JupyterLab profiles when 
+you log in. 
+A user JupyterLab profiles provides a particular combination of pre-installed 
+package versions and resource settings. 
+
+If you have already started your session and need to change JupyterLab profiles, 
+you can do this by selecting ‘Hub control panel’ from the ‘File’ menu within
+JupyterLab. Then click the ‘Stop my server’ button and wait for your current
+server to shut down. When the ‘Start my server’ button appears, you can click
+on it to return to the user JupyterLab profiles menu.
+
+### Logging out
+
+To log out, select ‘Log out’ from the ‘File’ menu within JupyterLab. 
+
+Note that your JupyterLab session will continue in the background even after
+you have logged out, but will eventually be terminated due to inactivity. 
+If you wish to stop your session explicitly,
+you can use the hub control panel as described in the 
+[Choosing a JupyterLab profile](#choosing-a-jupyterlab-profile) section above.
+
+
+
+# DeepESDL xcube viewer
+
+The DeepESDL xcube viewer is reachable via 
+https://viewer.earthsystemdatalab.net/ .
+
+![img.png](../img/xcube-viewer.png)
+
+The viewer currently shows public datasets, and later will allow to see 
+team/user cubes when logged in. The login will be via GitHub as for the DeepESDL
+JupyterLab. 
+
+For the functionality of the viewer, please refer to the documentation:
+https://xcube.readthedocs.io/en/latest/viewer.html#functionality
+
+
+# DeepESDL datacube resources
+
+DeepESDL provides a growing list of relevant variables for Earth System Science. 
+Most of them have been derived from Earth Observation, but the compilation also 
+includes model or re-analysis data if deemed useful.
+DeepESDL is very grateful to all data owners for kindly providing the data sets 
+free of charge. They come without any warranty, neither from the owners, 
+the DeepESDL team or ESA.
```

Comment:
All datacubes generated and distributed by DeepESDL come without any warranty, neither from the owners, from the DeepESDL, nor form ESA.